.github/workflows/release.yml

```diff
@@ -69,6 +78,44 @@ jobs:
             github.rest.git.createRef({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              ref: 'refs/tags/lambda-v' + "${{ env.VERSION }}",
+              ref: 'refs/tags/' + "${{ steps.version.outputs.tag }}",
               sha: context.sha
             })
+
+  regular-sar:
+    if: ${{ needs.release.outputs.enabled == 'true' }}
+    runs-on: ubuntu-latest
+    needs: release
+    permissions:
+      # The OIDC permissions can be found at https://github.com/elastic/oblt-infra/tree/main/conf/resources/repos/elastic-serverless-forwarder
+      id-token: write
+      contents: read
+    env:
+      BUCKET_NAME : "elastic-serverless-forwarder"
+      AWS_REGION : "eu-central-1"
+      # elastic-observability-prod
+      AWS_ACCOUNT_ID: "267093732750"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.release.outputs.tag }}
+
+      - uses: elastic/oblt-actions/aws/auth@v1
+        with:
+          aws-account-id: "${{ env.AWS_ACCOUNT_ID}}"
+          aws-region: "${{ env.AWS_REGION }}"
+
+      - uses: aws-actions/setup-sam@2360ef6d90015369947b45b496193ab9976a9b04 # v2
+        with:
+          use-installer: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and package
+        run: |
+          .internal/aws/scripts/dist.sh \
```

Comment:
It runs the same command in https://github.com/elastic/elastic-serverless-forwarder/issues/279#issuecomment-2305126967